customer-config/pnp/pnp.edge-synchronizer-config.v1.json

```diff
@@ -51,18 +51,18 @@
             "ITEM_IDENTIFIER",
             "PROMOTION",
             "PRICE_SPECIFICATION",
-            "ASSORTMENT_POLICY",
-            "DEPOSIT_RULE",
-            "EXTERNAL_DEVICE_ITEM",
-            "PRINT_COMMAND",
-            "ENTITY_CHANGES",
-            "ENTITY_VALIDATION_RESULT"
+            "DEPOSIT_RULE"
           ]
         },
         "synchronized": {
           "description": "If true, this entity will be synchronized by the Edge Synchronizer client.",
           "type": "boolean",
           "default": false
+        },
+        "isTenantwideEntity": {
+          "description": "If true, this entity will considered as not being specific to a business unit or a business unit group.",
```

Comment:
Let's put this description: `If true, this entity will be considered as global for a specific tenant and not defined per business unit or business unit group.`